content/id/docs/tasks/configure-pod-container/share-process-namespace.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Pembagian Namespace Process antara Kontainer-Kontainer pada sebuah Pod
```

Comment:
Can we translate into "Namespace Proses"?
Also, "Container" is an API name, do we need to translate? if not,  we change all "kontainer" into "Container".
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/
```suggestion
title: Pembagian Namespace Proses antar Container pada sebuah Pod
```

Reply:
+1 for this